### ISSUE TITLE
Horizontal scroll not working with virtualization

### ISSUE THREAD

**Issue by ashu-g** [react-component/tree#575]
When I enable virtualization, my auto horizontal scroll does not work. What I think might be the root cause is the 'rc-virtual-list' is listening to 'wheel' event which is not handling horizontal scroll. Even if I move my touchpad, left/right, it scrolls up and down only.
Anyone faced this issue? Quick help appreciated!

**Comment by ashu-g** [comment]
Any ideas here?

**Comment by ashu-g** [comment]
checking on this again please!

**Comment by cong1223** [comment]
yes, It doesn't support sub-features, and I think it can be done

**Comment by cong1223** [comment]
> 这里有什么想法吗？
Did you solve it？

**Comment by cong1223** [comment]
> 这里有什么联想吗？

Did you solve it？

**Comment by agusmakmun** [comment]
The only possible now for both scrollable by adding custom style, and without `itemHeight`.

```jsx
height={300}
virtual={true}
style={{width: '200px'}}
```

**Comment by tomzacchia** [comment]
@agusmakmun do you have a working example? When `itemHeight` prop is not defined it doesn't virtualize the node list.
